src/Microsoft.AspNetCore.Sockets.Client/LongPollingTransport.cs

```diff
@@ -20,12 +21,10 @@ public class LongPollingTransport : ITransport
 
         private readonly HttpClient _httpClient;
         private readonly ILogger _logger;
-        private readonly CancellationTokenSource _senderCts = new CancellationTokenSource();
-        private readonly CancellationTokenSource _pollCts = new CancellationTokenSource();
-
-        private IPipelineConnection _pipeline;
+        private IChannelConnection<Message> _application;
         private Task _sender;
         private Task _poller;
+        private readonly CancellationTokenSource _transportCts = new CancellationTokenSource();
 
         public Task Running { get; private set; }
```

Comment:
No, it isn't. It was there and I left it because it is useful for testing but it's a bit weird.